css/responsive/desktop.styl

```diff
@@ -1,12 +1,15 @@
-$header-height = 90px
-$footer-height = 90px
+$header-height = 60px
+$footer-height = 48px
 
 // BASIC
 
 .main-content-padding
   padding-top: $header-height
   padding-bottom: $footer-height
 
+.min-height-content 
+  min-height: calc('100vh - ' + ($header-height * 2))
```

Comment:
its cause the header is always taller than the footer. if i set it to `100vh - (header + footer)` then the content won't actually be centered, it will be slightly too low.